### DISCUSSION TITLE
Is that possible to have variable

### DISCUSSION THREAD

**Discussion by duongthaiha** [microsoft/promptflow#3089] [Q&A]
Hi
I am trying find a way to make a variable available for the duration of the flow. ie A global variable that can be appended from all the node. The idea here is to capture the log in each nodes please.
Thank you very much in advance.

**Comment by 0mza987** [comment]
Hi @duongthaiha ,

Prompt flow does not have a global variable that can be modified by all nodes. According to your scenario, think you can pass a local file path to the flow as a flow input, and then pass it to all nodes that needs to write logs, inside a node you can modify the file to append your logs. Does that work for you?
